### PR TITLE
With reqwest

### PR DESCRIPTION
**Pull request by badboy**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Switch from `curl` & `rustc_serialize` to `reqwest` & `serde`
+
+This makes it far more future proof and additionally does not require
+openssl on Mac or Windows!
+
+# v1.0.0 (2016-02-02)
+
+The missing `after_all_success` hook for Travis
+
+Did you ever wnated to run a task after all jobs of your build finished and succeeded?
```
Comment (schultyy) [inline CHANGELOG.md]: wnated -> wanted

**Comment by badboy** [comment]
bors try

**Comment by badboy** [comment]
bors try

**Pull request merged**